gui/default/syncthing/device/changeLogModalView.html

```diff
@@ -0,0 +1,26 @@
+<style> th, td { padding: 6px; } </style>
+<modal id="globalDiskChanges" status="default" icon="{{'history'}}" heading="{{'Global Changes' | translate}}" large="yes" closeable="yes">
+    <div class="modal-body">
+        <table>
+          <tr>
+            <th translate>Device</th>
+            <th translate>Action</th>
+            <th translate>Type</th>
+            <th translate>Path</th>
+            <th translate>Time</th>
+          </tr>
+          <tr ng-repeat="changeEvent in globalChangeEvents">
+            <td>{{deviceName(findDevice(changeEvent.data.modifiedBy))}}</td>
```

Comment:
The config wrapper should be able to provide a `DeviceFromShort(id ShortID) DeviceConfiguration` or similar based on an internal lookup table. The short ID to long ID mapping is guaranteed to be unique per config.